src/Enum/EnumTrait.php

```diff
@@ -101,9 +101,8 @@ trait EnumTrait
 
     /* --- EXCEPTIONS --- */
 
-    protected static function throwInvalidMemberException(string $member): void
+    private static function throwInvalidMemberException(string $member): void
```

Comment:
subclass of a class extending any `AbstractXEnum` would not be able to customize its exceptions